worlds/models/circuit/model.sdf

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<sdf version="1.4">
+    <model name="circuit">
+        <static>true</static>
+
+        <link name="link">
+            <collision name="collision">
+                <geometry>
+                    <plane>
+                        <normal>0 0 1</normal>
+                        <size>1000 1000</size>    <!-- Since is ground, make collision bigger than visual -->
```

Comment:
other places you use `100 x 100`, I personally think `1000 x 1000` is excessive for a robot of our scale (10s of centimeters)